privacy-on-beam/pbeam/count_test.go

```diff
@@ -285,13 +336,16 @@ func TestCountWithPartitionsCrossPartitionContributionBounding(t *testing.T) {
 	p, s, col, want := ptest.CreateList2(pairs, result)
 	col = beam.ParDo(s, pairToKV, col)
 
-	partitions := []int{0, 1, 2}
-	partitionsCol := CreateList(s, partitions)
+	partitions := []int{0, 1, 2, 3, 4}
+	partitionsCol := beam.CreateList(s, partitions)
 
-	epsilon, delta, k, l1Sensitivity := 50.0, 0.01, 25.0, 3.0
+	// We have 5 partitions. So, to get an overall flakiness of 10⁻²³,
+	// we need to have each partition pass with 1-10⁻²⁵ probability (k=25).
+	epsilon, delta, k, l1Sensitivity := 50.0, 0.0, 25.0, 3.0
 	pcol := MakePrivate(s, col, NewPrivacySpec(epsilon, delta))
-	got := Count(s, pcol, CountParams{MaxPartitionsContributed: 3, MaxValue: 1, NoiseKind: LaplaceNoise{}, specifiedPartitions: partitionsCol})
-	// With a max contribution of 3, all of the data for partitions 0, 1, and 2 should be kept.
+	got := Count(s, pcol, CountParams{MaxPartitionsContributed: 3, MaxValue: 1, NoiseKind: LaplaceNoise{}, partitionsCol: partitionsCol})
+	// With a max contribution of 3, 40% of the data should be
```

Comment:
Nit: "40% of the data in the specified partitions"

It is a bit ambiguous if the 40% refers to the whole data or the specified partitions data.